src/test/sighash_tests.cpp

```diff
@@ -81,6 +81,9 @@ uint256 static SignatureHashOld(CScript scriptCode, const CTransaction& txTo, un
         txTmp.vin.resize(1);
     }
 
+    // Blank out the joinsplit signature.
+    memset(&txTmp.joinSplitSig[0], 0, 64);
```

Comment:
Or `txTmp.joinSplitSig.size()` since `sizeof(boost::array<unsigned char, 64>)` need not be 64, depending on the implementation of `boost::array`.